packages/flutter/lib/src/material/i18n/localizations.dart

```diff
@@ -204,7 +270,14 @@ const Map<String, Map<String, String>> localizations = const <String, Map<String
     "closeButtonTooltip": r"Close",
     "nextMonthTooltip": r"Next month",
     "previousMonthTooltip": r"Previous month",
+    "nextPageTooltip": r"Next page",
+    "previousPageTooltip": r"Previous page",
     "licensesPageTitle": r"Licenses",
+    "pageRowsInfoTitle": r"$firstRow–$lastRow of $rowCount",
+    "pageRowsInfoTitleApproximate": r"$firstRow–$lastRow of about $rowCount",
+    "rowsPerPageTitle": r"Rows per page:",
+    "selectedRowCountTitleOne": r"1 item selected",
```

Comment:
english should have a "selected count title zero" that says "No items" instead of "0 items"